frontend/src/components/Markdown/Markdown.tsx

```diff
@@ -0,0 +1,82 @@
+import ReactMarkdown from "react-markdown";
+import rehypeHighlight from "rehype-highlight";
+import "highlight.js/styles/atom-one-dark.css";
```

Comment:
Is it possible to make the theme dynamic depending on whether it's light or dark mode? Currently, in light mode the code is not very readable.

Reply:
Hello,Thank you for your reply. The issue should be resolved now, and I would appreciate it if you could review the code. As a beginner in programming, I'm open to any feedback on inconsistencies or areas for improvement. Please feel free to offer any advice or suggestions.

Reply:
![image](https://github.com/user-attachments/assets/7ca33fe7-4a57-497c-8da2-4f9d79907e5e)
It is better now

Reply:
Thanks! Just for consistency - Could you use `"highlight.js/styles/github.css"` for light mode so it is similar to dark mode's theme?

Reply:
OK,thank you for the great suggestion; I've made the changes.